### PR TITLE
Expose lint errors for a page/revision via a GET request.

### PR DESCRIPTION
**Pull request by Pchelolo**
Bug: T164006

cc @wikimedia/services @subbuss @arlolra

### CODE REVIEW

**Comment by subbuss** [comment]
LGTM.

File: v1/content.yaml [hunk 1]
```diff
@@ -612,6 +612,80 @@ paths:
...
+        options:
+          redirect_cache_control: '{{options.response_cache_control}}'
+      - path: lib/security_response_header_filter.js
+    get:
+      tags:
+        - Page content
+      summary: Get the linter errors for a specific title/revision.
+      description: |
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      operationId: getFormatRevision
```
Comment (d00rman) [inline v1/content.yaml]: c/p error? Also, no need for an operation id here.

File: v1/content.yaml [hunk 2]
```diff
@@ -612,6 +612,80 @@ paths:
                 x-restbase-parentrevision: '{{x-restbase-parentrevision}}'
       x-monitor: false
 
+  /lint/{title}{/revision}:
```
Comment (d00rman) [inline v1/content.yaml]: `lint` doesn't seem too informative to me. Perhaps `lintstatus` or `linterrors` ? Any suggestions @subbuss / @Pchelolo ?

Comment (Pchelolo) [reply v1/content.yaml]: `lint` was chosen for consistency with the transform endpoint `transform/wikitext/to/lint`...

Comment (d00rman) [reply v1/content.yaml]: Oh, excellent point @Pchelolo

File: v1/content.yaml (hunk 1 above)
Comment (Pchelolo) [reply v1/content.yaml]: Done

**Pull request merged**